src/ucp/tag/eager_snd.c

```diff
@@ -48,30 +60,30 @@ static size_t ucp_tag_pack_eager_dt(void *dest, void *arg)
     ucp_request_t *req = arg;
     size_t length;
 
-    length         = req->send.length;
```

Comment:
IMHO we need to rename ucp_tag_pack_eager_dt() to ucp_tag_pack_eager_**single**_dt()